protocol/contracts/reserve/src/error.rs

```diff
@@ -0,0 +1,26 @@
+use thiserror::Error;
+
+use sdk::cosmwasm_std::StdError;
+
+#[derive(Error, Debug, PartialEq)]
+pub enum ContractError {
+    #[error("[Reserve] [Std] {0}")]
+    Std(#[from] StdError),
+
```

Comment:
I'm not sure about all these empty lines but I feel like this either isn't set in the `rustfmt` configuration or we have problems with it reading said configuration.

Reply:
the [config option](https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#blank_lines_upper_bound) is still unstable